lib/console.js

```diff
@@ -35,26 +35,25 @@ function Console(stdout, stderr) {
   }
 }
 
-Console.prototype.log = function() {
-  this._stdout.write(util.format.apply(null, arguments) + '\n');
+Console.prototype.log = function(...args) {
+  this._stdout.write(`${util.format(...args)}\n`);
```

Comment:
FWIW v8 5.x supposedly has improved this significantly (according to the v8 blog), but we will have to test and see how it compares there.

Also, the other day I happened to notice that backtick strings are much slower to create than typical concatenation. I'm not sure what, if any, impact it has on GC though.